scalableminds/webknossos
# Redesign user task list

This PR replaces the user task list table with a list of "cards". This makes it easier to read the whole description of a task and doesn't break long project/task-type names. Let's wait for feedback and refine accordingly

### URL of deployed dev instance (used for testing):
- https://redesignusertasklist.webknossos.xyz

### Steps to test:
- Create and get some tasks, so the user task list is populated.
- Check whether the user task list is rendered correctly, the description is fully readable and the actions still work.
- The list with the finished tasks and the admin view for a users tasks (Users - Show Tracings) should look the same, but the description should be restricted in height, to allow for better readability when scanning many tasks.

### Issues:
- fixes #2817

------
